cms/djangoapps/contentstore/views/videos.py

```diff
@@ -285,33 +286,44 @@ def convert_video_status(video):
     return status
 
 
-def _get_videos(course):
+def _get_videos(course, request):
     """
     Retrieves the list of videos from VAL corresponding to this course.
     """
-    videos = list(get_videos_for_course(course.id, VideoSortField.created, SortDirection.desc))
+    params = {
+        "page": request.GET.get("page", 1),
+        "page_size": request.GET.get("page_size", 20),
```

Comment:
On 2nd thought we should not specify default page size here. It should be defined in Val.